sanic/sanic.py

```diff
@@ -231,6 +232,17 @@ def converted_response_type(self, response):
                     response = HTTPResponse(
                         "An error occured while handling an error")
 
+            if self.debug:
+                error_body = response.body
+                if isinstance(error_body, bytes):
+                    try:
+                        error_body = error_body.decode()
+                    except:
+                        print(
```

Comment:
Would it be better to log this this as an error instead of printing it to stderr?